src/neo/Ledger/Blockchain.cs

```diff
@@ -429,7 +429,9 @@ private void OnParallelVerified(ParallelVerified parallelVerified)
             RelayResultReason reason = parallelVerified.VerifyResult;
             if (reason == RelayResultReason.Succeed)
             {
-                if (!MemPool.CanTransactionFitInPool(parallelVerified.Transaction))
+                if (ContainsTransaction(parallelVerified.Transaction.Hash))
```

Comment:
Do we need to really check the "Mempool" of just the view?
Perhaps we could skip the first line of this method.

Reply:
Yes, you are right. In this case no need to recheck mompool, just check the view is enough.

Reply:
I just modified, @eryeer, maybe it will be enough